docs/index.md

```diff
@@ -0,0 +1,36 @@
+# Content Cache Backends Config Operator
+
+NOTES: This charm works with the a WIP content-cache charm rewrite. This charm does not work with the current content-cache charm.
+
+<!-- A single sentence that says what the product is, succinctly and memorably. Add a 1-2 sentence description of what the charm software does. -->
+
+This charm is a subordinate charm to the Content Cache charm. This charm should be integrated with the Content Cache charm to inject the charm configuration of this charm. For more information see the [Content Cache charm](https://charmhub.io/content-cache).
```

Comment:
The description sounds a bit weird, I'd rather explain that this charm allows to provide to content-cache charm the configuration required to expose a backend service behind content-cache enabling caching capabilities

Reply:
Updated the description.